# Single-cell-multi-omics-profiling-of-ccRCC
Single-cell multi-omics profiling reveals the comprehensive epigenetic regulatory features of clear cell renal cell carcinoma

operating systems: Linux; 64vCPU; 256GiB memory

R version 4.0.2 – R environment for all custom code
Signac 1.0.0 – scATAC-seq data analysis, Cell Clustering Implementation, Matching TF Motifs within peak regions
Seurat 4.0.0 – scRNA-seq data analysis, Cell Clustering Implementation
Harmony 1.0 – Eliminate batch effect
RNA velocity 0.6 - Estimate the direction and rate of differentiation from cells
SeuratWrappers - 0.2.0 Integrate seurat-related algorithms
GenomeInfoDb 1.24.2 - Genomic Ranges Operations used for overlap analyses
GenomicRanges 1.40.0 - Genomic Ranges Operations used for overlap analyses
Monocle3 1.0.0 - Reconstruct the cell fate decisions 
infercnv 1.4.0 - CNV analysis
CellPhoneDB 0.24 - Ligand–receptor interactions of cells
Beyondcell 1.3.3 - Predict conventional drugs sensitive to ccRCC
EnsDb.Hsapiens.v75 2.99.0 - Gene reference library
JASPAR2020 0.99.10 - Getting Genomic DNA sequences for motif matching and footprinting
TFBSTools 1.26.0 – Getting Genomic DNA sequences for motif matching and footprinting
chromVAR 1.10.0 - Motif analysis, calculating TF deviation scores which can be associated with TF activity
samtools 1.10 – For Bulk ATAC-seq data analysis
bedtools 2.27.1 - For Bulk ATAC-seq data analysis
bamCoverage 3.1.1 - For Bulk ATAC-seq data analysis
IGV 2.11.9 - a software Visualizing Bulk ATAC-seq results
Proteome Discoverer 2.2 - a software Analysis of protein mass spectrometry results
