# Single-cell-multi-omics-profiling-of-ccRCC
Single-cell multi-omics profiling reveals the comprehensive epigenetic regulatory features of clear cell renal cell carcinoma

operating systems: Linux; 64vCPU; 256GiB memory

R version 4.0.2 – R environment for all custom code

R packages:
Signac 1.0.0 – scATAC-seq data analysis, Cell Clustering Implementation, Matching TF Motifs within peak regions
Seurat 4.0.0 – scRNA-seq data analysis, Cell Clustering Implementation
Harmony 1.0 – Eliminate batch effect
velocyto.R 0.6 - Estimate the direction and rate of differentiation from cells
pagoda2 1.0.7 - Estimate the direction and rate of differentiation from cells
SeuratWrappers - 0.2.0 Integrate seurat-related algorithms
GenomeInfoDb 1.24.2 - Genomic Ranges Operations used for overlap analyses
GenomicRanges 1.40.0 - Genomic Ranges Operations used for overlap analyses
Monocle3 1.0.0 - Reconstruct the cell fate decisions 
infercnv 1.4.0 - CNV analysis
tidyverse 1.3.0 - 1.3.0
Beyondcell 1.3.3 - Predict conventional drugs sensitive to ccRCC
EnsDb.Hsapiens.v75 2.99.0 - Gene reference library
JASPAR2020 0.99.10 - Getting Genomic DNA sequences for motif matching and footprinting
TFBSTools 1.26.0 – Getting Genomic DNA sequences for motif matching and footprinting
chromVAR 1.10.0 - Motif analysis, calculating TF deviation scores which can be associated with TF activity
magrittr 1.5 - scRNA-seq data analysis
cowplot 1.0.0 - scRNA-seq data analysis
dplyr 1.0.2 - scRNA-seq data analysis
motifmatchr 1.10.0 - Motif analysis
patchwork 1.0.1 - expands the API to allow for arbitrarily complex composition of plots by, among others
BSgenome.Hsapiens.UCSC.hg38 1.4.3 - Full genome sequences for Homo sapiens (Human)
ggplot2 3.3.2 - plot
psych 1.8.12 - Analysis of the relationship between ligand receptors
qgraph 1.9 - Analysis of the relationship between ligand receptors
igraph 1.2.5 - Analysis of the relationship between ligand receptors
tidyverse 1.3.0 - Analysis of the relationship between ligand receptors
scales 1.1.1 - Analysis of the relationship between ligand receptors
ggsci 2.9 - Analysis of the relationship between ligand receptors
reshape2 1.4.4 - Analysis of the relationship between ligand receptors
DT 0.15 - Analysis of the relationship between ligand receptors
ggthemes 4.2.4 - Analysis of the relationship between ligand receptors
tidyr 1.1.2 - Analysis of the relationship between ligand receptors
ComplexHeatmap 2.4.3 - Analysis of the relationship between ligand receptors
circlize 0.4.12 - Analysis of the relationship between ligand receptors

Linux software:
CellPhoneDB 0.24 - Ligand–receptor interactions of cells
samtools 1.10 – For Bulk ATAC-seq data analysis
bedtools 2.27.1 - For Bulk ATAC-seq data analysis
bamCoverage 3.1.1 - For Bulk ATAC-seq data analysis

